qiita_pet/handlers/study_handlers/description_handlers.py

```diff
@@ -188,12 +188,19 @@ def process_sample_template(self, study, user, callback):
         msg_level = "success"
         is_mapping_file = looks_like_qiime_mapping_file(fp_rsp)
 
+        if is_mapping_file and data_type == "":
```

Comment:
Is `data_type` only ever a `str`? If so, then this can be reduced to `if is_mapping_file and not data_type`